GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -968,11 +1078,34 @@ private String sanitize(String s) {
                                                 .replaceAll("\n", "")
                                                 .trim();
                             }
+
                             createCell(
                                     row,
                                     offset++,
                                     header,
                                     headerStyle);
+
+                            // check if we need to create columns for all options
+                            if (QuestionType.OPTION == q.getType()  && useQuestionId) {
```

Comment:
useQID includes the fact that there is a non-null question identifier. However, we don't use this in the creation of the column names, so we don't care about that. In fact, we need to use useQuestionId here.